frontend/app/components/task-selection/template.hbs

```diff
@@ -48,6 +48,13 @@
         optionTemplate=this.taskOptionTemplate
       )
     )
+    comment=(component
+      (ensure-safe-component "comment-input")
+      placeholder="Comment"
+      value=this.comment
+      onChange=this.onCommentChange
+      class="form-control comment-field"
+    )
```

Comment:
I think the `comment-input` should be part of the `filter-sidebar` component, instead of the `task-selection`. It's more of a *filter* and the `task-selection` is used in various places. Also makes the whole initialization stuff easier, as you don't have to struggle with the `task-selection` component (which is *horrifying IMHO*)... :)

Reply:
i moved it to `filter-sidebar/filter` https://github.com/adfinis/timed/pull/283/commits/89f2bf1eaf68795e16bc54fe3b22ead5d1e14d15#diff-cdc879d2c92851bda53f78bd96507f73ea47da51b6f9c084e65bc0ca4fe09d48

Reply:
Sorry for being so picky here, but I think this splintered component (previously `comment-input`) is not well maintainable. :sweat_smile: 

I wanted to suggest decoupling the whole comment search input from the `task-selection` component. Just like you started out in the beginning, with your `comment-input`, but instead of loading it into the task-selection component why not directly use it in the `analytics/index` template like we do for the [user select](https://github.com/adfinis/timed/commit/89f2bf1eaf68795e16bc54fe3b22ead5d1e14d15#diff-12c469691b8d9581a646f849b70514cb098e3442b584edecf96d2b6f9ac75079R60)? 

The `task-selection` component should really not become any more bloated as it already is.

So:
* (re-)add `comment-input` or whatever you wanna call it
* make direct use of it in `analysis/index` template
* keeps `filters-sidebar/filter` and `task-selection` untouched --> less :spaghetti: more :sparkles:

Reply:
yea i agree, i changed it now